nifi-nar-bundles/nifi-kafka-bundle/nifi-kafka-shared/src/main/java/org/apache/nifi/kafka/shared/property/SaslMechanism.java

```diff
@@ -31,6 +31,8 @@ public enum SaslMechanism implements DescribedValue {
 
     PLAIN("PLAIN", "PLAIN", "Plain username and password authentication"),
 
+    OAUTHBEARER("OAUTHBEARER", "OAUTHBEARER", "Login with Azure Active Directory"),
```

Comment:
As we only support AAD oauh token here so it should better to rename it to AADOAUTHBEARER